server/routes/createGoal/createGoalController.test.ts

```diff
@@ -69,7 +70,9 @@ describe('createGoalController', () => {
         'targetDate-year': '2024',
         action: 'submit-form',
       }
-      req.session.addStepForms = []
+      req.session.newGoalForm = {
```

Comment:
`req.session.newGoalForm` or `req.session.newGoal`? 🤔 

I guess it depends on how we define a 'form' object?

Reply:
I think you're right - `req.session.newGoal` .... I'll change it 👍